tracer/build/smoke_test_snapshots/smoke_test_iis_snapshots.json

```diff
@@ -0,0 +1,134 @@
+[
+  [
+    {
+      "trace_id": 4102954709290435042,
```

Comment:
Just curious since I don't know, so for this one we don't need to scrub the ids nor any other attribute or use generic values?

Reply:
Exactly - the [dd-apm-test-agent](https://github.com/DataDog/dd-apm-test-agent) takes care of that. IIRC I _did_ scrub some of the values previously, just to make that clearer, but I was lazy here 🙈